### PR TITLE
Docs for `preferred_chains` issuer and global config

### PR DESCRIPTION
**Pull request by Klooven**
Added docs for the `preferred_chains` option that was added to Caddyfile.

Ref. caddyserver/caddy#4192

### CODE REVIEW

File: src/docs/markdown/caddyfile/directives/tls.md [hunk 1]
```diff
@@ -134,6 +138,10 @@ Obtains certificates using the ACME protocol.
 - **trusted_roots** is one or more root certificates (as PEM filenames) to trust when connecting to the ACME CA server.
 - **dns** configures the DNS challenge.
 - **resolvers** customizes the DNS resolvers used when performing the DNS challenge; these take precedence over system resolvers or any default ones.
+- **preferred_chains** specifies which certificate chains Caddy should prefer; useful if your CA provides multiple chains. Use **one** of the following options:
+	- **smallest** will tell Caddy to prefer chains with the fewest amount of bytes
+	- **root_common_name** is a list of one or more common names; Caddy will choose the first chain that has a root that matches with at least one of the specified common names
+	- **any_common_name** is a list of one or more common names; Caddy will choose the first chain that has an issuer that matches with at least one of the specified common names
```
Comment (mholt) [inline src/docs/markdown/caddyfile/directives/tls.md]:
```suggestion
- **preferred_chains** specifies which certificate chains Caddy should prefer; useful if your CA provides multiple chains. Use one of the following options:
	- **smallest** will tell Caddy to prefer chains with the fewest amount of bytes.
	- **root_common_name** is a list of one or more common names; Caddy will choose the first chain that has a root that matches with at least one of the specified common names.
	- **any_common_name** is a list of one or more common names; Caddy will choose the first chain that has an issuer that matches with at least one of the specified common names.
```

File: src/docs/markdown/caddyfile/options.md [hunk 2]
```diff
@@ -166,6 +170,14 @@ Defines the issuer (or source) of TLS certificates. The tokens following the nam
 ##### `ocsp_stapling`
 Can be set to `off` to disable OCSP stapling. Useful in environments where responders are not reachable due to firewalls.
 
+##### `preferred_chains`
+If your CA provides multiple certificate chains, you can use this option to specify which chain Caddy should prefer. Set **one** of the following options:
+
+- **smallest** will tell Caddy to prefer chains with the fewest amount of bytes
+- **root_common_name** is a list of one or more common names; Caddy will choose the first chain that has a root that matches with at least one of the specified common names
+- **any_common_name** is a list of one or more common names; Caddy will choose the first chain that has an issuer that matches with at least one of the specified common names
```
Comment (mholt) [inline src/docs/markdown/caddyfile/options.md]:
```suggestion
##### `preferred_chains`
If your CA provides multiple certificate chains, you can use this option to specify which chain Caddy should prefer. Set one of the following options:

- **smallest** will tell Caddy to prefer chains with the fewest amount of bytes.
- **root_common_name** is a list of one or more common names; Caddy will choose the first chain that has a root that matches with at least one of the specified common names.
- **any_common_name** is a list of one or more common names; Caddy will choose the first chain that has an issuer that matches with at least one of the specified common names.
```

**Review by mholt — CHANGES_REQUESTED** [review]
LGTM, thanks for completing your contribution with docs! Just a couple minor formatting/punctuation nits.

**Comment by Klooven** [comment]
@mholt Should be fixed now 🙂

**Review by mholt — APPROVED** [review]
Perfection

**Pull request merged**